src/mux-analytics.brs

```diff
@@ -516,18 +539,29 @@ function muxAnalytics() as Object
     end if
   end sub
 
+  prototype.videoPositionChangeHandler = sub(position as Double)
```

Comment:
How often does this trigger? Does this trigger frequently enough to give us accurate playhead position tracking?